Gopkg.lock

```diff
@@ -1,6 +1,17 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "development"
+  digest = "1:ad77b20c92f2418802e50880d27084689d742fe263735b239aff52a713720627"
+  name = "github.com/ChainSafe/gossamer"
```

Comment:
is this not part of Gopkg.toml already ?

Reply:
Good catch, that sneaked in because of a local experiment